pkg/stanza/fileconsumer/file.go

```diff
@@ -196,7 +196,7 @@ func (m *Manager) makeFingerprint(path string) (*fingerprint.Fingerprint, *os.Fi
 		return nil, nil
 	}
 
-	if len(fp.FirstBytes) == 0 {
+	if fp.BytesLength == 0 {
```

Comment:
should we replace by an "Empty" method instead?

Reply:
Is that considered to be cleaner?

Reply:
yes, you are leaking implementation details to consuming code.